.travis.yml

```diff
@@ -18,13 +19,31 @@ matrix:
             services:
                 - memcached
         -
+            sudo: required
+            php: 7.2
+            env:
+                - SYLIUS_SUITE="application"
+                - TRAVIS_NODE_VERSION="7.5"
+            services:
+                - memcached
+        -
+            sudo: required
+            php: nightly
```

Comment:
I'd skip the nightly PHP version for now as there's still some time until PHP 7.3 gets released & builds will take more time to finish.

Reply:
Instead, running packages build on PHP 7.2 is a must :)

Reply:
Mainly added nightly for now as it might discover some issues now already that we can tackle directly. Will take it out before finalising PR.